src/api/test/functional/webui/patchinfo_create_test.rb

```diff
@@ -3,136 +3,56 @@
 
 class Webui::PatchinfoCreateTest < Webui::IntegrationTest
 
-  uses_transaction :test_create_patchinfo_with_too_short_summary
-  uses_transaction :test_create_patchinfo_with_too_short_sum_and_desc
+  LONG_DESCRIPTION = "long description" * 15
 
   setup do
     use_js
-    @project = 'home:Iggy'
+    login_Iggy
+    visit project_show_path(project: "home:Iggy")
   end
 
-  def open_new_patchinfo
-    click_link("Create patchinfo")
-    page.must_have_text "Patchinfo-Editor for "
+  teardown do
+    login_Iggy
```

Comment:
True. It just felt weird to have a login call at the end of that test. I'll change it.